### PR TITLE
Adding custom analyzers sample for incoming tutorial

### PR DESCRIPTION
**Pull request by dereklegenzoff**

### CODE REVIEW

File: custom-analyzers/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - azure
+  - azure-cognitive-search
+urlFragment: custom-analyzer-tutorial
+---
+
+# Create a custom analyzer in Azure Cognitive Search using REST APIs and Postman
+
+![Flask sample MIT license badge](https://img.shields.io/badge/license-MIT-green.svg)
+
+This Postman collection creates a basic search index and builds an analyzer designed for searching phone numbers. The collection and corresponding tutorial walk through the process of testing sample searches, testing how those searches are analyzed, and building an analyzer to intuitively search phone number.
```
Comment (HeidiSteen) [inline custom-analyzers/README.md:18]: nit:  numbers should be plural

File: custom-analyzers/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+This collection is featured in the [Tutorial: Create a custom analyzer using REST APIs](https://docs.microsoft.com/en-us/azure/search/tutorial-create-custom-analyzer). When you import the collection, modify the headers and URL to use your service name and API key.
```
Comment (HeidiSteen) [inline custom-analyzers/README.md:20]: in the link, remove en-us

File: custom-analyzers/README.md (hunk 1 above)
Comment (dereklegenzoff) [reply custom-analyzers/README.md]: Great catch - thanks! Updated.

File: custom-analyzers/README.md (hunk 2 above)
Comment (dereklegenzoff) [reply custom-analyzers/README.md]: Thanks! I removed en-us

**Pull request merged**